orbit/cmd/orbit/orbit.go

```diff
@@ -151,9 +151,15 @@ func main() {
 			Usage:   "Launch Fleet Desktop application (flag currently only used on darwin)",
 			EnvVars: []string{"ORBIT_FLEET_DESKTOP"},
 		},
+		// Note: this flag doesn't have any effect anymore. I'm keeping
+		// it just for backwards compatibility since some users were
+		// using it because softwareupdated was causing problems and I
+		// don't want to break their setups.
+		//
+		// For more context please check out: https://github.com/fleetdm/fleet/issues/11777
 		&cli.BoolFlag{
 			Name:    "disable-kickstart-softwareupdated",
-			Usage:   "Disable periodic execution of 'launchctl kickstart -k softwareupdated' on macOS",
+			Usage:   "(Deprecated) Disable periodic execution of 'launchctl kickstart -k softwareupdated' on macOS",
```

Comment:
any thoughts on keeping this? is there anything we could to to make it clear that this flag doesn't have an effect anymore?

Reply:
Is it not enough to just be loud about it in the release notes?

If we're going to keep it, is there way to surface a warning/notification to a command-line user when the flag is set? And then we can remove it in a future version?

Reply:
printing a warning is a good idea! I will add that

I don't think being loud in the release notes is enough because orbit auto-updates, and it won't be able to start anymore after an update if we completely remove the flag. (I have tried)